first-order-odes/integrating-factor.md

```diff
@@ -155,3 +155,79 @@ c(t) &= \frac{A}{k}
 \end{align}
 
 ````
+
+## Skill Builder
+```{problem}
```

Comment:
You are not getting a preview on this PR because there is no `{problem}` directive. Take a look at some of the other skill builders to see how this should be done!
```suggestion
```